BEPs/BEP-525.md

```diff
@@ -22,32 +22,37 @@
       - [Vote v1](#vote-v1)
       - [Transactions v1](#transactions-v1)
   - [4. Backwards Compatibility](#4-backwards-compatibility)
+    - [4.1 Validator Operator](#41-validator-operator)
+    - [4.2.RPC Providers And Other Full Nodes Operators](#42rpc-providers-and-other-full-nodes-operators)
+    - [4.3 MEV](#43-mev)
   - [5. License](#5-license)
 
 ## 1. Summary
-Add a network layer, which accept validator or validator authorized node to join in to improve the network efficiency and reduce network latency between validators.
+It is a new network topology, which only accepts validators or validator authorized nodes tp join and transactions in this network will be forwarded directionally to a few validators that are most likely to produce the next block.
 
 ## 2. Motivation
-Low latency is one of the key user experience and if BSC wanna to support shorter block interval, network latency between validators could be crucial to support sub-second block interval. Current P2P gossip based network is good at broadcasting blocks/transactions to the whole network, but it is not efficient enough, i.e. it would take lots network bandwidth and also cost non-negligible latency. Validators are the key player to sustain the network, so one validator dedicated network layer could help to address the two issues.
+- For low network latency between validators: transaction latency is one of the key user experience. As BSC targets to short its block interval to sub-second, it is quite crucial to reduce the network latency between validators. Current P2P gossip based network is good at broadcasting blocks/transactions to the whole network, but it is not efficient enough, i.e. it would take lots of network bandwidth and also cost non-negligible latency.
```

Comment:
```
transaction latency is one of the key user experience.
```
I think we could add a new paragraph here to describe the modification of the transaction.